README.md

```diff
@@ -43,6 +43,12 @@ Lanelet2 has been released for ROS. Just install `ros-[distribution]-lanelet2`,
 sudo apt install ros-noetic-lanelet2
 ```
 
+### Without ROS
+Outside of ROS, Lanelet2 can be installed from PyPI. Note that currently only Python 3.8 builds are available.
```

Comment:
```suggestion
### Python
For usage in python, Lanelet2 can be installed from PyPI. Note that currently only python 3.8 linux builds are available.
```

Reply:
I would rather explicitly distinguish between usage inside and outside of ROS. With ROS you still use Python, but you don't use `pip`.
Regarding capitalization: on python.org Python is always capitalized. I'll add the linux modifier though.

Reply:
That's true, I meant the `python-only` nature of it. So "without ROS" is also "without c++". But adding linux was the more important part 👍